documentation/cli/docs/02_wallet.md

```diff
@@ -89,23 +89,23 @@ When just initialised, the wallet has no account yet, use the `new` command to c
 
 #### Examples
 
-Initialise the wallet with a randomly generated mnemonic and the default node.
```

Comment:
I think there is a general consensus that documentation should be American english. And we have an inconsistent use so it can't hurt to harmoniZe a bit.